STTBot/events/app_mention.py

```diff
@@ -302,7 +302,15 @@ def _cmd_pin_remove(client, event_data, command, say):
     data_interface.remove_pin(permalink.channel, permalink.timestamp)
     return {"message": ":white_check_mark: Successfully removed pin"}
 
-
+def _STT_draft(client, event_data, command, say):
+    if len(command.args) == 0;
+        raise CommandError("No list provided")
+    else:
+        draft_order = command.args
+        random.shuffle(draft_order)
+        
+    return {"message": ":robot_face: Order generated:", draft_order}  
```

Comment:
This is invalid dictionary syntax, the draft order should be part of the second string, i.e.

`{"message": f":robot_face: Order generated: {draft_order}"}`

However, draft_order is a list so it won't embed into the string particularly nicely by default. Look into the `join` string function to see how to convert a list to a string.

Reply:
hopefully now resolved